account-kit/smart-contracts/src/ma-v2/account/account.ts

```diff
@@ -0,0 +1,127 @@
+import type {
+  EntryPointDef,
+  SmartAccountSigner,
+  SmartContractAccountWithSigner,
+  ToSmartContractAccountParams,
+} from "@aa-sdk/core";
+import {
+  createBundlerClient,
+  getAccountAddress,
+  getEntryPoint,
+  toSmartContractAccount,
+} from "@aa-sdk/core";
+import {
+  concatHex,
+  encodeFunctionData,
+  type Address,
+  type Chain,
+  type Hex,
+  type Transport,
+} from "viem";
+import { MAV2FactoryAbi } from "../abis/maV2Factory.js";
+import { addresses } from "../addresses.js";
+import { standardExecutor } from "../../msca/account/standardExecutor.js";
+import { multiOwnerMessageSigner } from "../../msca/plugins/multi-owner/signer.js"; // TODO: swap for MA v2 signer
+
+export const DEFAULT_OWNER_ENTITY_ID = 0;
+
+export type SMAV2Account<
+  TSigner extends SmartAccountSigner = SmartAccountSigner
+> = SmartContractAccountWithSigner<"SMAV2Account", TSigner, "0.7.0">;
+
+export type CreateSMAV2AccountAccountParams<
```

Comment:
did you mean `CreateSMAV2AccountParams` here?

Reply:
good catch!